gtsam/hybrid/DCGaussianMixtureFactor.h

```diff
@@ -69,6 +67,28 @@ class DCGaussianMixtureFactor : public Factor {
     return 0;
   }
 
+  // TODO(dellaert): Does not make sense
+  double error(const gtsam::Values& continuousVals,
+               const gtsam::DiscreteValues& discreteVals) const override {
+    return 0;
```

Comment:
throw?

Reply:
Agreed: `throw std::runtime_error("DCGaussianMixtureFactor::error not implemented");`